README.md

```
@@ -117,7 +117,7 @@ Meet our wonderful team comprised of Product Managers, Designers, Tech Leads, En
             </td>
             <td align="center" width="150">
                 <a href="https://umd.hack4impact.org/">
-                <img src="public/team-photos/pending-image.jpg" height="100" width="100" style="border-radius:50%;object-fit:cover;"/><br/>
+                <img src="public/team-photos/shreyas-thirumale.jpg" height="100" width="100" style="border-radius:50%;object-fit:cover;"/><br/>
```

markdownlint-cli2 (0.20.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)